curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-and-share-about-educational-and-professional-background/657b160d6a8662610fe6a523.md

```diff
@@ -0,0 +1,53 @@
+---
+id: 657b160d6a8662610fe6a523
+title: Task 22
+challengeType: 22
+dashedName: task-22
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Candidate2: Hello! Certainly. I worked at Mock Corporation for five years. I was part of a team of software engineers, and we were responsible for developing and maintaining various applications.
+-->
+
+# --description--
+
+The simple past tense in English is used for actions or situations that were completed in the past. Regular verbs in the past tense usually end with `-ed`, while irregular verbs have unique past forms. For example, as you learned in previous tasks, `was` and `were` are the past forms of `is` and `are`.
+
+Examples of simple past tense:
+
+- `I worked at a company.` (`work` becomes `worked`)
+- `I was happy.` (`is` becomes `was`)
+- `We were busy.` (`are` becomes `were`)
```

Comment:
```suggestion
Examples of simple past tense:

`I worked at a company.` (`work` becomes `worked`)
`I was happy.` (`is` becomes `was`)
`We were busy.` (`are` becomes `were`)
```

Reply:
It should look like this the following, right?

> Examples of simple past tense
> 
> `I worked at a company.` (`work` becomes `worked`)
> `I was happy.` (`is` becomes `was`)
> `We were busy.` (`are` becomes `were`)